multigeiger/multigeiger.ino

```diff
@@ -681,8 +685,10 @@ int jb_HV_gen_charge__chargepulses() {
     digitalWrite(PIN_HV_FET_OUTPUT, LOW);               // turn the HV FET off
     delayMicroseconds(1000);
     chargepulses++;
-  } while ((chargepulses < 1000) && !isr_GMC_cap_full); // either a timeout or a capacitor full interrupt stops this loop
+  } while ((chargepulses < max_chargepulses) && !isr_GMC_cap_full); // either a timeout or a capacitor full interrupt stops this loop
   time2hvpulse = millis();                              // we just pulsed, so restart timer
+  if((chargepulses == max_chargepulses) && !isr_GMC_cap_full)
+    Serial.println("Error: HV charging failed!");       // pulsed a lot, but still the capacitor is not at desired voltage
```

Comment:
just a quickfix here to not let the error be completely silent.

for users not having connected the serial terminal it of course would be better to also update some global "hv_good" state and display it on the oled display.